controllers/comments.controller.js

```diff
@@ -8,4 +8,21 @@ exports.getCommentsByArticleId = (req, res, next) => {
             res.status(200).send({ comments });
         })
         .catch(next); 
+};
+
+exports.postCommentToArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
+      return res.status(400).send({ msg: "Please provide both username and body in your request." });
+  }
```

Comment:
If the body is invalid when you make the db.query PSQL will throw an error, it is better to catch this err in the catch block and send it to an middleware error handler in the app than to declare custom errors in every function.

Refer to the notes on PSQL errors to see how to do it